drivers/usb/udc/udc_mcux_ehci.c

```diff
@@ -676,11 +716,19 @@ static int udc_mcux_set_address(const struct device *dev, const uint8_t addr)
 
 static int udc_mcux_enable(const struct device *dev)
 {
+	struct udc_mcux_data *priv = udc_get_private(dev);
+
+	udc_mcux_change_state(dev, 1, priv->vbus_present);
```

Comment:
udc_mcux_enable() should not emit UDC_EVT_VBUS_READY because there is no reason to do it at this point.